R/summaries.R

```diff
@@ -368,7 +368,13 @@ setMethod("countDominantFeatures", signature = c(x = "SummarizedExperiment"),
 #'
 #' @export
 setMethod("summary", signature = c(object = "SummarizedExperiment"),
-    function(object, assay_name = abund_values, abund_values = "counts"){
+    function(object, assay_name = NULL, abund_values = NULL){
+        # Specify assay names for user
+        message("Following assays detected: ", call. = FALSE)
+        print(assayNames(object))
```

Comment:
I'm wondering if this is useful information. Furthermore, because it is printing, user cannot just remove this info when disabling messages

Reply:
`print` should not be used in functions; it should be with `message` always

Reply:
@microsud I noticed the comments are still pending. If you could add the requested change we can merge.

Reply:
Ok I am now fixing this in another PR to this branch..